vmdb/app/models/miq_provision_vmware/customization.rb

```diff
@@ -266,7 +266,6 @@ def set_spec_array_option(obj, property, key, override_value=nil)
     if key.nil?
       value = get_option(nil, override_value)
     else
-      value = override_value.nil? ? get_option(key) : override_value
       value = get_option(key)
```

Comment:
I agree this is a typo and we will want to remove ```value = get_option(key)``` and keep the first one that honors the override value being passed in.  This might actually resolve an issue I have run into before.  Nice find.